lib/parallel-fields-extractor.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (elasticSearchResponse) => {
+  elasticSearchResponse.hits.hits.forEach((bib) => {
+    if (bib._source.parallelDisplayField) {
+      bib._source.parallelDisplayField.forEach((field) => {
+        const capitalizedFieldName = field.fieldName.charAt(0).toUpperCase() + field.fieldName.slice(1)
```

Comment:
Are we sure that the parallel field name and the field name are always related in this way? I think the mapping between fields and parallel fields is set in nypl core. I'd prefer to use that, as it is more authoritative, if it isn't massively more complicated.

Reply:
Yes. They're always related in this way. That's also how the front-end will need to relate them.